README.md

```diff
@@ -32,7 +32,7 @@ Available on [jsdelivr](https://cdn.jsdelivr.net/npm/vue-resource@1.5.1), [unpkg
     // get body data
     this.someData = response.body;
 
-  }, response => {
+  }, error => {
```

Comment:
The error callback is for http and non-http errors, so the respose object in this context is still relevant.